src/viewer.js

```diff
@@ -546,89 +539,97 @@ class VolumeImageViewer {
     }
 
     // Metadata Tiles types checks for each instance
-    // look for channels
-    for (let i = 0; i < this[_options].metadata.length; ++i) {
-      const instanceMetadata = formatMetadata(this[_options].metadata[i])
-      if (instanceMetadata.SamplesPerPixel !== 1) {
-        // this is not a monochorme channel, but a color image.
-        continue
-      }
-      if (instanceMetadata.DimensionOrganizationType === '3D' || instanceMetadata.DimensionOrganizationType === '3D_TEMPORAL') {
-        // 3D data
-        // TO DO: get some example data.
-        throw new Error('Volume Image Viewer does hot hanlde 3D channel data yet.')
-      } else if (instanceMetadata.DimensionOrganizationType === 'TILED_FULL') {
-        if (instanceMetadata.TotalPixelMatrixFocalPlanes !== 1) {
-          continue
-        } else {
-          const pathIdentifier = instanceMetadata.OpticalPathSequence[0].OpticalPathIdentifier
-          const channel = this[_channels].find(channel => {
-            return channel.blendingInformation.opticalPathIdentifier === pathIdentifier
-          })
-          if (channel) {
-            channel.addMetadata(this[_options].metadata[i])
+    // group channels by OpticalPathIdentifier
```

Comment:
We cannot just group by Optical Path Identifier (see https://github.com/MGHComputationalPathology/slim/issues/22#issuecomment-853365144)

Reply:
Or we need to assert that each instance has a unique identifier. But what do we do if that's not the case? The best may be to introduce a Optical Path UID to the standard (see https://github.com/MGHComputationalPathology/slim/issues/22#issuecomment-853369764)

Reply:
yes I agree, I will not apply/close this comment now, but I will apply it as soon as we figure out the the grouping issue in MGHComputationalPathology/slim#22